docs/Reference/architecture/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# CONTRIBUTING
+
+## Installation
+
+Cosmos Cash diagrams are based on PlantUML. 
+
+PlantUML can be installed on your local system through your platforms' package manager:
+
+MacOS, through [Homebrew](https://brew.sh/)
+
+```
+> brew install plantuml
+```
+
+Windows, through [Chocolatey](https://chocolatey.org/)
+
+```
+> choco install plantuml
+```
+
+Linux-based systems, through your distributions package manager
+
+## Contributing diagrams
+
+The source code for all diagrams SHOULD be stored in `./src` folder AND have the `.puml` suffix. Please refer to the [PlanUML documentation](https://plantuml.com) for further details.
+
+## Themes
+
+Where possible all diagrams will be rendered to have a similar look and feel. To achieve this add the following to your diagram:
```

Comment:
```suggestion
Where possible all diagrams are rendered to have a similar look and feel. To achieve this look and feel, add the following theme to your diagram:
```

Reply:
now that we're out of the ADR (requires future tense, per Cosmos SDK convention) let's use standards for present tense, per https://developers.google.com/style/tense